client-telemetry-reporter-plugin/kafka/README.md

```diff
@@ -0,0 +1,209 @@
+<!-- title: How to use KIP-714 for centralized monitoring of Kafka Clients -->
+<!-- description: In this tutorial, learn how to use KIP-714 for centralized monitoring of Kafka Clients. -->
+
+# How to use KIP-714 for centralized monitoring of Kafka Clients
```

Comment:
suggest not including "KIP-714" in the title. Just "How to centrally monitor Kafka clients via broker configuration" or something

Reply:
Hmmm, I went with your suggestion but was thinking that searches will be for `kip-714` then how can we optimize the tutorial to show up. I have added `kip-714` in page description.